### PR TITLE
V2.0 changes, breaks API backwards compatibility.

### PR DESCRIPTION
**Pull request by matthewpersico**

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Copyright 2017 Chris Carpita
```
Comment (ccarpita) [inline LICENSE.md:1]: I've labelled as MIT license recently (see README) to clarify rights for contributors, so this file should be consistent.  Thanks for adding though!

**Comment by matthewpersico** [comment]
I added the file explicitly because there are systems that look for it in
order to automatically figure out if it will be allowed in house at
commercial shops. Like mine. :-)

File: kvstore.sh [hunk 2]
```diff
@@ -331,3 +347,9 @@ kvstore () {
 }
 kvstore "$@"
 exit $?
+
+## for emacs...
+## Local Variables:
+## sh-basic-offset: 2
+## sh-indentation: 2
+## End:
```
Comment (ccarpita) [inline kvstore.sh]: Is it possible for emacs to read these settings on a project/directory level?   Would prefer to keep editor-specific code out, but then again you're the biggest contributor, so we can keep it if it needs to be file-level.

**Comment by ccarpita** [comment]
@matthewpersico understood about the license file, which is a good idea.  I realize now that it is the MIT license in fact.  No need to change anything 👍

File: kvstore.sh (hunk 2 above)
Comment (matthewpersico) [reply kvstore.sh]: Yes. I did some research and found that I could add a .dir-locals.el file to the directory and stuff those items in there. I'll amend the commit this evening,

**Comment by ccarpita** [comment]
That solution sounds good to me

**Comment by matthewpersico** [comment]
Done. Emacs config moved out to .dir-locals.el.

**Comment by ccarpita** [comment]
👍

**Pull request merged**